src/Mapping/Table.php

```diff
@@ -21,5 +22,24 @@ public function __construct(
         public readonly array|null $uniqueConstraints = null,
         public readonly array $options = [],
     ) {
+        if ($this->indexes !== null) {
+            Deprecation::trigger(
+                'doctrine/orm',
+                'https://github.com/doctrine/orm/pull/11353',
```

Comment:
Should be 11357

Reply:
Changed.

Reply:
Not everywhere it seems :wink:

Reply:
Oh gosh shame on me :see_no_evil:! Sorry for another iteration of this :laughing:

Reply:
No worries :slightly_smiling_face: